portal/src/AppRoot.tsx

```diff
@@ -34,7 +34,7 @@ const AppRoot: React.FC = function AppRoot() {
   }, [appID]);
   return (
     <ApolloProvider client={client}>
-      <ScreenLayout>
+      <ScreenLayout showScreenNav={true}>
```

Comment:
I think we should not be touching AppRoot and ScreenLayout. Can we follow the structure of `AppsScreen`?